package.json

```diff
@@ -1,28 +1,29 @@
 {
   "name": "@shoutem/ui",
-  "version": "0.22.0-alpha.2",
+  "version": "0.22.0",
   "description": "Styleable set of components for React Native applications",
   "dependencies": {
-    "@shoutem/animation": "~0.11.0",
-    "@shoutem/theme": "~0.10.0",
+    "@shoutem/animation": "~0.12.0",
+    "@shoutem/theme": "~0.11.0",
     "babel-plugin-transform-decorators-legacy": "~1.3.4",
     "buffer": "~4.5.1",
     "events": "1.1.0",
     "html-entities": "~1.2.0",
     "htmlparser2": "~3.9.0",
     "lodash": "~4.15.0",
     "qs": "~4.0.0",
-    "react-native-lightbox": "shoutem/react-native-lightbox#4e58b9b1aca166142d69b161ff511efc3358ff9f",
+    "react-native-lightbox": "shoutem/react-native-lightbox#master",
```

Comment:
Don't point to master directly, add a tag to both repos and point to tag. This might get broken if someone commits to master.